types/angular/jqlite.d.ts

```diff
@@ -651,6 +651,7 @@ interface JQuery {
 
     // Undocumented
     length: number;
+    [index: number]: TElement;
```

Comment:
Unfortunately that was the first thing I tried. It worked fine isolation (in my project that doesn't have `jquery/index.d.ts`), but it raises the following error when I try to compile it in the DT project:

```
'Interface 'JQuery<TElement>' incorrectly extends interface 'ArrayLike<TElement>'.
  Index signatures are incompatible.
    Type 'HTMLElement' is not assignable to type 'TElement'.'
at: '28,11'
source: 'ts'
```
Unfortunately I'm going to have to leave this for now. I think the workaround in the meantime is to load the full jQuery typings if you need this API in your project.